test/jdk/javax/print/PrintServiceLookup/CountPrintServices.java

```diff
@@ -25,14 +25,14 @@
 import java.io.InputStreamReader;
 import javax.print.PrintService;
 import javax.print.PrintServiceLookup;
-import javax.print.attribute.AttributeSet;
-import javax.print.attribute.HashAttributeSet;
-import javax.print.attribute.standard.PrinterName;
+
+import jtreg.SkippedException;
 
 /*
  * @test
  * @bug 8032693
  * @key printer
+ * @library /test/lib/
```

Comment:
GitHub isn't letting me click to comment on the line I want to but this test looks like another one that should require(os == linux)

Reply:
Thanks for the suggestion. OS type check in code has been replaced as `@requires (os.family == "linux")`